lib/safira/accounts/attendee.ex

```diff
@@ -15,6 +15,7 @@ defmodule Safira.Accounts.Attendee do
     field :volunteer, :boolean, default: false
     field :avatar, Safira.Avatar.Type
     field :name, :string
+    field :token_balance, :integer, default: 0
     field :discord_association_code, Ecto.UUID, autogenerate: true
```

Comment:
Now that I'm noticing: was this branch created from the branch from PR #153 instead of from the master branch? It seems like it, which makes it a bit confusing to know what are the real changes implemented in this PR.